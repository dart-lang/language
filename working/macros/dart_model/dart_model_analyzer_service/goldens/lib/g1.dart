abstract class Foo {
  abstract int bar;
}

class Baz {
  int get x => 0;
  final int foo = 3;
}
