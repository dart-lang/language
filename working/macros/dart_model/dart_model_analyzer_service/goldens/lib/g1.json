{
  "uris": {
    "package:goldens/g1.dart": {
      "Foo": {
        "properties": {
          "class": true,
          "abstract": true
        },
        "annotations": [],
        "supertype": "dart:core/object.dart#Object",
        "interfaces": [],
        "members": {
          "bar": {
            "properties": {
              "abstract": true,
              "getter": false,
              "method": false,
              "field": true,
              "static": false,
              "synthetic": false
            }
          }
        }
      },
      "Baz": {
        "properties": {
          "class": true,
          "abstract": false
        },
        "annotations": [],
        "supertype": "dart:core/object.dart#Object",
        "interfaces": [],
        "members": {
          "foo": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": false,
              "field": true,
              "static": false,
              "synthetic": false
            }
          },
          "x": {
            "properties": {
              "abstract": false,
              "getter": true,
              "method": false,
              "field": false,
              "static": false,
              "synthetic": false
            }
          }
        }
      }
    },
    "dart:core": {
      "Object": {
        "properties": {
          "class": true,
          "abstract": false
        },
        "annotations": [
          {
            "type": "dart:core/annotations.dart#pragma",
            "value": {
              "options": null,
              "name": "vm:entry-point"
            }
          }
        ],
        "interfaces": [],
        "members": {
          "hashCode": {
            "properties": {
              "abstract": false,
              "getter": true,
              "method": false,
              "field": false,
              "static": false,
              "synthetic": false
            }
          },
          "runtimeType": {
            "properties": {
              "abstract": false,
              "getter": true,
              "method": false,
              "field": false,
              "static": false,
              "synthetic": false
            }
          },
          "==": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": false,
              "synthetic": false
            }
          },
          "toString": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": false,
              "synthetic": false
            }
          },
          "noSuchMethod": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": false,
              "synthetic": false
            }
          },
          "hash": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": true,
              "synthetic": false
            }
          },
          "hashAll": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": true,
              "synthetic": false
            }
          },
          "hashAllUnordered": {
            "properties": {
              "abstract": false,
              "getter": false,
              "method": true,
              "field": false,
              "static": true,
              "synthetic": false
            }
          }
        }
      }
    }
  }
}
